### DISCUSSION TITLE
Parental Search Lock.

### DISCUSSION THREAD

**Discussion by vroman374** [TeamNewPipe/NewPipe#11037] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Hello All! 

Firstly, I would like to thank all involved in the making and maintaining of this app!

My question is: Is there a way to have a Parental Search lock on this app? I have young children and would love to download this app on all their tablets. Yet,  I fear them having Open access to anything that may pop up in their search.  
I had an app some years ago that allowed you to add just the channels you wanted to watch and had a password for the Search so you couldn't not add anything else to the app. 

Is that possible here?
Thank you all again!

Vinnie Roman NJ

**Comment by opusforlife2** [comment]
Have you tried https://www.f-droid.org/en/packages/free.rm.skytube.oss/ ?
